cmd/fdroidcl/show.go

```diff
@@ -47,11 +48,37 @@ func findApps(ids []string) []*fdroidcl.App {
 	apps := appsMap(mustLoadIndexes())
 	result := make([]*fdroidcl.App, len(ids))
 	for i, id := range ids {
-		app, e := apps[id]
-		if !e {
-			log.Fatalf("Could not find app with ID '%s'", id)
+		s := strings.Index(id, "=")
+		if s > -1 {
+			ver, err := strconv.Atoi(id[s+1:])
+			if err != nil {
+				log.Fatalf("Could not parse version code from '%s'", id)
+			}
+			id = id[:s]
+
+			app, e := apps[id]
+			if !e {
+				log.Fatalf("Could not find app with ID '%s'", id)
+			}
+
+			found := false
+			for _, apk := range app.Apks {
+				if apk.VCode == ver {
+					app.Apks = []fdroidcl.Apk{apk}
+					found = true
+				}
+			}
+			if !found {
+				log.Fatalf("Could not find version %d for app with ID '%s'", ver, id)
+			}
+			result[i] = app
+		} else {
+			app, e := apps[id]
+			if !e {
+				log.Fatalf("Could not find app with ID '%s'", id)
+			}
+			result[i] = app
```

Comment:
This bit of code is now duplicated. You should split the logic so that it isn't.